src/components/data-visualization/ChartDemoGroup.js

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+
+import * as ChartComponents from '@carbon/charts-react';
+
+import H2 from 'gatsby-theme-carbon/src/components/markdown/H2';
+import P from 'gatsby-theme-carbon/src/components/markdown/P';
+import CodeBar from '../ComponentDemo/Code/CodeBar.js';
+
+import {
+  CardGroup,
+  MiniCard,
+} from 'gatsby-theme-carbon/src/components/MiniCard';
+
+import STORYBOOK_LOGO from '../../images/storybook-icon.svg';
+
+const generateHeadingID = (title) =>
+  title
+    .split(' ')
+    .map((t) => t.toLowerCase())
+    .join('-');
+
+const ChartDemoGroup = ({ demoGroup, light }) => {
+  let demo = demoGroup.demos.find((d) => d.mainDemo === true);
+  if (!demo) {
+    demo = demoGroup.demos[0];
+  }
+
+  const DemoComponent = ChartComponents[demo.chartType.vanilla];
+
+  const numberOfRemainingDemos =
+    demoGroup.demos.length > 2 ? demoGroup.demos.length - 1 : 0;
+
+  return (
+    <>
```

Comment:
You probably just want to use a proper `div` here so the page's stack styles get applied if you inspect elements on the other page you can kinda see what's happening:

<img width="267" alt="Screen Shot 2021-04-13 at 1 56 18 PM" src="https://user-images.githubusercontent.com/4078018/114605770-0a633780-9c60-11eb-9f7c-e31ee1a8cf0d.png">

Modifying the `--space` css custom property either through react style prop or css would be the preferred way to override the stack spacing behavior.